demo-shell/karma.conf.js

```diff
@@ -50,7 +50,7 @@ module.exports = function (config) {
         colors: true,
         logLevel: config.LOG_INFO,
         autoWatch: true,
-        browsers: ['Chrome'],
+        browsers: [/*'Chrome',*/ 'ChromeHeadless'],
```

Comment:
Why the comment?

Reply:
Removed, just followed the style from `process-services/karma.config.js`